flink-ml-lib/src/main/java/org/apache/flink/ml/common/datastream/DataStreamUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.common.datastream;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+
+/** Provides utility functions for {@link org.apache.flink.streaming.api.datastream.DataStream}. */
```

Comment:
nits: It is not very common to use full path in the comment like this. How about simplifying it to `Provides utility functions for {@link DataStream}`

Reply:
The citation is generated automatically. There seems a error if I use `{@link DataStream}`.

BTW, Flink has an example for this: `org.apache.flink.streaming.api.operators.Output`.